.github/workflows/ci.yml

```
@@ -9,92 +9,65 @@
 name: ci
 
 on:
+  push:
+    branches: [ develop ]
```

GH actions for commits pushed to these branches (you can keep develop if nessesary)

```suggestion
    branches:
      - master
      - release/*
```